python/instrumentation/openinference-instrumentation-groq/pyproject.toml

```diff
@@ -0,0 +1,91 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "openinference-instrumentation-groq"
+dynamic = ["version"]
+description = "OpenInference Groq Instrumentation"
+readme = "README.md"
+license = "Apache-2.0"
+requires-python = ">=3.8, <3.13"
+authors = [
+    { name = "OpenInference Authors", email = "oss@arize.com" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+dependencies = [
+    "opentelemetry-api",
+    "opentelemetry-instrumentation",
+    "opentelemetry-semantic-conventions",
+    "openinference-instrumentation>=0.1.7",
+    "openinference-semantic-conventions",
+    "wrapt",
+    "typing-extensions",
+]
+
+
+[project.optional-dependencies]
+instruments = [
+    "groq >= 0.9.0",
+]
+test = [
+    "groq >= 0.9.0",
```

Comment:
`we can pin the dependency at its lower bound, like [this](https://github.com/Arize-ai/openinference/blob/6fe05914b2b9584cb43fbe5408ac2780256ffae8/python/instrumentation/openinference-instrumentation-vertexai/pyproject.toml#L41), because there’s actually a [second](https://github.com/Arize-ai/openinference/blob/6fe05914b2b9584cb43fbe5408ac2780256ffae8/python/tox.ini#L44) test in tox.ini where install -U is done to test the latest version (this is why sometimes CI fails for another package you didn’t even touch. it’s basically acting as a canary) (edited)` - roger